problem_builder/tests/unit/test_problem_builder.py

```diff
@@ -71,76 +76,65 @@ def setUp(self):
-    )
-    def test_theme_uses_default_theme_if_no_mentoring_theme_is_set_up(self, theme_config):
-        self.service_mock.get_settings_bucket = Mock(return_value=theme_config)
-        self.assertEqual(self.block.get_theme(), _default_theme_config)
-        self.service_mock.get_settings_bucket.assert_called_once_with(self.block)
-
-    @ddt.data(
-        {MentoringBlock.theme_key: 123},
-        {MentoringBlock.theme_key: [1, 2, 3]},
-        {MentoringBlock.theme_key: {'package': 'qwerty', 'locations': ['something_else.css']}},
-    )
-    def test_theme_correctly_returns_configured_theme(self, theme_config):
-        self.service_mock.get_settings_bucket = Mock(return_value=theme_config)
-        self.assertEqual(self.block.get_theme(), theme_config[MentoringBlock.theme_key])
-
-    def test_theme_files_are_loaded_from_correct_package(self):
-        fragment = MagicMock()
-        package_name = 'some_package'
-        theme_config = {MentoringBlock.theme_key: {'package': package_name, 'locations': ['lms.css']}}
-        self.service_mock.get_settings_bucket = Mock(return_value=theme_config)
-        with patch("problem_builder.mentoring.ResourceLoader") as patched_resource_loader:
-            self.block.include_theme_files(fragment)
-            patched_resource_loader.assert_called_with(package_name)
-
-    @ddt.data(
-        ('problem_builder', ['public/themes/lms.css']),
-        ('problem_builder', ['public/themes/lms.css', 'public/themes/lms.part2.css']),
-        ('my_app.my_rules', ['typography.css', 'icons.css']),
-    )
-    @ddt.unpack
-    def test_theme_files_are_added_to_fragment(self, package_name, locations):
-        fragment = MagicMock()
-        theme_config = {MentoringBlock.theme_key: {'package': package_name, 'locations': locations}}
-        self.service_mock.get_settings_bucket = Mock(return_value=theme_config)
-        with patch("problem_builder.mentoring.ResourceLoader.load_unicode") as patched_load_unicode:
-            self.block.include_theme_files(fragment)
-            for location in locations:
-                patched_load_unicode.assert_any_call(location)
-
-            self.assertEqual(patched_load_unicode.call_count, len(locations))
-
     def test_student_view_calls_include_theme_files(self):
+        self.service_mock.get_settings_bucket = Mock(return_value={})
         with patch.object(self.block, 'include_theme_files') as patched_include_theme_files:
             fragment = self.block.student_view({})
             patched_include_theme_files.assert_called_with(fragment)
 
     def test_author_preview_view_calls_include_theme_files(self):
+        self.service_mock.get_settings_bucket = Mock(return_value={})
         with patch.object(self.block, 'include_theme_files') as patched_include_theme_files:
             fragment = self.block.author_preview_view({})
             patched_include_theme_files.assert_called_with(fragment)
 
 
+@ddt.ddt
+class TestMentoringBlockOptions(unittest.TestCase):
```

Comment:
I think it would have been good to leave in one or two theming tests. I know the logic is moved to xblock-utils and tested there, but if someone else changes that API we need to know.